spec/request/api/v1/auth/sign_out_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require 'faker'
+
+describe 'Auth', type: :request do
+  let!(:user) { create(:user) }
+  let(:sign_in_params) do
+    {
+      email: user.email,
+      password: user.password
+    }
+  end
+
+  before do
+    post api_v1_user_session_path sign_in_params
```

Comment:
Isn't necessary to sign in a user doing a post before each test. Just send the headers on the delete request

`user.create_new_auth_token` or using a helper if you already have it.